css/_uswds-theme-custom-styles.scss

```diff
@@ -428,6 +429,9 @@ img {
       @include u-bg($site-background-color);
       margin: units(2) units(2) units(4);
     }
+    @include at-media("widescreen") {
+      pointer-events: all;
+    }
```

Comment:
MDN Docs labels `pointer-events: all` as an experimental, SVG-only value. While it _does_ seem to work here. I wonder if we should change it to `pointer-events: auto` as a more consistent option to if we never set it's parent element to `pointer-events: none`

> **Note**
> `pointer-events: auto`  The element behaves as it would if the pointer-events property were not specified. [^1]

[^1]: https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events

Reply:
Resolved in [a04c3eb](https://github.com/uswds/uswds-site/pull/2046/commits/a04c3ebad6a91408925ae830264cc72a3f66184d)